src/components/sectionList/listActions/DefaultListActions.tsx

```diff
@@ -9,12 +9,14 @@ type DefaultListActionProps = {
     model: ModelWithAccess
     onShowDetailsClick: (model: ModelWithAccess) => void
     onOpenSharingClick: (id: string) => void
+    onDeleteClick: () => void
```

Comment:
Think it's more versatile to pass the model here. Should probably add `displayName` to default filter fields and thus make it always available in BaseListModel.

Then you can specify the name of what you're deleting in a confirmation modal

Reply:
> Think it's more versatile to pass the model here. Or we could handle that in the parent.

I'd be in favor of letting the parent handle model-related logic. I think we should keep this component unaware of the model. The only reason we're passing the model is to assess whether the user can edit this item. We'd make this component and `SectionListRow` quite a bit simpler if we moved the handling of the concept "model" to the `SectionListWrapper` and then also stop passing the model to DefaultListActions and SectionListRow:

* The `SectionListWrapper` is supplying the callbacks and is already aware of the model, so passing it to `DefaultListActions` and then threading it back to the parent component is adding two avoidable levels of indirection
* Both the SectionListRow and the DefaultListActions wouldn't have to call the `canEditModel` function, narrowing down the scope of the props while reducing duplication and coupling
* The callback signatures of both the `SectionListRow` as well as the `DefaultListActions` would be simpler, making the components more versatile and re-usable in different contexts (if that's ever necessary :man_shrugging:)
* Even `renderActions` and `renderColumnValue` could be supplied with the model in `SectionListWrapper`, removing the need to pass the model to `SectionListRow` entirely. We'd remove a few levels of indirection here two, centralizing the concept "model" into a single component

Reply:
> Should probably add displayName to default filter fields and thus make it always available in BaseListModel.
> Then you can specify the name of what you're deleting in a confirmation modal

You mean adding that to the query that's provided by `useModelListView` and run in the `List.tsx` components?

Reply:
> > Should probably add displayName to default filter fields and thus make it always available in BaseListModel.
> > Then you can specify the name of what you're deleting in a confirmation modal
> 
> You mean adding that to the query that's provided by `useModelListView` and run in the `List.tsx` components?

Add it here https://github.com/dhis2/maintenance-app-beta/blob/master/src/lib/sectionList/fieldFilters.ts . Then it will be fetched for every model

Reply:
Added `displayName` and showing it in the alerts